bedrock/firefox/templates/firefox/developer.html

```diff
@@ -93,6 +93,28 @@
 <section class="feature-overview">
   <div class="container">
     <ul class="features">
+    {% if LANG.startswith('en-') %}
```

Comment:
@flodolo The bug mentions this is en-US only. I asked about it in the bug because I thought perhaps it could be en-* so, still waiting on that. If it will be translated later on, then I reckon using a tag would be better.

@alexgibson I reckon that would be a good idea yes.